weekly-standup-template.md

```diff
@@ -79,16 +74,36 @@ Other issues can be dealt with in coworking time.
 
 ### Notes
 
-- Summary of discussion points
-- Use subsections if appropriate
+#### Blockers
+
+- Blocker
+  - Address the blocker
+
+#### Coworking session suggestions
+
+- Suggestion
+- Suggestion
 
-### Decisions
+#### Plans for the week
 
-- Record any decisions made during the meeting
+<!--
+Please use the template below to report what you expect or want to work on this week.
+
+Please include full details for stories or issues and include links.
+-->
+
+<!--
+##### Your name here
+
+- Work on Story Z
+- by addressing issues X
+-->
 
 ### Actions
 
-| Owner             | Action                               |
-| -------           | --------                             |
-| Meeting organiser | Open meeting record for next meeting |
-| Meeting organiser | Save meeting record (sharepoint)     |
+| Owner                              | Action                                          |
+| -------                            | --------                                        |
+| Meeting organiser                  | Open meeting record for next meeting            |
+| Meeting organiser                  | Save meeting record (sharepoint)                |
+| Technical lead and project manager | Compare actual effort against project board     |
+| Technical lead and project manager | Compare plan for the week against project board |
```

Comment:
Yes, my thinking here is to 'save' non-management people from having to listen to this discussion and keep this meeting lean by doing it elsewhere.

It might mean we want a regular 'managers' meeting to do this.